utils/economic-checks/Cargo.toml

```diff
@@ -59,6 +57,5 @@ lazy-pages = [
     "pallet-gear/lazy-pages",
 ]
 debug-wasm-mutate = []
-gear-native = ["gear-runtime", "sp-consensus-aura", "authoring-aura"]
+gear-native = ["gear-runtime", "sp-consensus-babe"]
 vara-native = ["vara-runtime", "sp-consensus-babe"]
```

Comment:
we can now remove optional usage of babe

Reply:
done